app/components/tooltip.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import RNToolTip from 'react-native-tooltip';
+
+export let isToolTipShowing = false;
```

Comment:
This helps to determine if there is a ToolTip component showing already in order to prevent going to a different screen when tapping outside to tooltip, is done this way to avoid unnecessary re-renders in every post instead of just setting it in redux

Reply:
I wonder if we could just declare `this.isToolTipShowing = false` and add a static function inside the component like:

```
    static getToolTipShowing() {
        return this.isToolTipShowing;
    }
```
And then have it imported and called at`post.js`

Reply:
That won't work cause is going to be defined for every component.

I could create a utils function with a `setToolTipVisible` and a `getToolTipVisible` so it is easier to understand

Reply:
Thanks for looking into it!